scripts/test-harness-contracts.ps1

```
@@ -2605,6 +2605,53 @@ try {
+# scoped re-review, escalation by failure, session effort as the floor
+# =====================================================================
+# The per-lens counters doubled the worst case to 6 rounds per task, and the
+# universal maximum-effort mandate priced every routine dispatch as if it were
+# the final filter. These assertions pin the proportional contract so a future
+# review round does not silently reintroduce either.
+
+# -- (a) One budget per task, shared by both lenses; per-lens counters are gone --
+Check-Match '.claude/skills/pelizzai-execute/references/task-cycle.md' '3 fix.{0,3}re-review cycles PER TASK[\s\S]{0,120}shared by both\s+lenses' 'task-cycle §5: the fix budget is per task and shared by both lenses'
+Check-NotMatch '.claude/skills/pelizzai-execute/references/task-cycle.md' 'PER LENS, PER TASK|separate counters' 'task-cycle §5: the per-lens counters do not come back'
+Check-Match '.claude/skills/pelizzai-execute/references/task-cycle.md' 'Passing one lens does not refund rounds already spent' 'task-cycle §5: a lens pass does not refund the shared budget'
+Check-Match '.claude/skills/pelizzai-execute/SKILL.md' '3 cycles per task, one budget shared\s+by both lenses' 'execute mirrors the shared per-task budget'
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Check-Match' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)